assignment-01.md

```diff
@@ -4,27 +4,45 @@ E.g. Question <br>
 {Answer} {Reference}
 
 1. What is the difference between private limited and public limited company? State one example for each type.
+Private Limited Company refers to the company which is not listed on a stock exchange and the shares are held privately by the members concerned while Public Limited Company implies a company that is listed on a recognized stock exchange and whose shares are traded openly by the public.
+Reference: https://keydifferences.com/difference-between-public-company-and-private-company.html
 
 2. Why is Berjaya Sports Toto Berhad registered under Non-Shariah Compliant?
+It is because it is related to gambling which is under Non-Shariah Compliant Business Activities.
 
 3. List down 3 Non-Shariah Compliant Business Activities. 
+Alcohol, Gambling, Pornography 
+Reference: https://av.sc.com/my/content/docs/Non-Syariah-Compliant-Business-Activities_ref.pdf
 
 4. Name the stock exchange of Malaysia.
+Kuala Lumpur Stock Exchange (KLSE)
 
 5. Create a new folder name 'AR' in your _repo main branch_. Upload Annual Report 2018 - 2020 for companies below: <br>
 (a) ViTrox Corp <br>
 (b) Pentamaster Corp <br> 
 *There should be at least 6 PDF files. <br>
+(a）https://www.vitrox.com/investor/annual-report.php
+(b) https://pentamaster.listedcompany.com/ar.html
+
 
 6. Who are the users of annual reports?
+Current shareholders and potential investors
+https://www.inc.com/encyclopedia/annual-reports.html
 
 7. What is the most important part of an annual report?
+The financial statements are the most important part of the annual report.
+https://www.freshbooks.com/hub/reports/write-an-annual-report
 
 8. You have RM 10,000 savings and want to purchase shares as an investment for earning side income. 
 You have listed two potential companies and need to select one company to invest. 
 Comparing _Pentamaster Corporation Bhd._ and _ViTrox Corporation Bhd._ Justify your investment choice.
+I will choose to invest in ViTrox Corporation Bhd. This is because their revenue and net income is keep increasing annually and have great performance in these 3 years (2019-2021). Which means this company still has big growing space. Meanwhile, Pentamaster Corporation Bhd. one is like up and down, which is not stable.
```

Comment:
Good attempt. There are a lot of factors that should be considered and **Financial analysis** will help you justify the decision better. Worry not, you will learn it from AKW104!